components/rmrk/Gallery/GalleryItem.vue

```diff
@@ -197,6 +198,8 @@ export default class GalleryItem extends mixins(PrefixMixin) {
   public message = ''
   public priceChartData: [Date, number][][] = []
   public showNavigation = false
+  public userBalance
+  public isBuyDisabled = true
```

Comment:
- missing types
- I would prefer `hasBuyDisabled`

Reply:
@roiLeo once I add types, eslint gives this error

`Type boolean trivially inferred from a boolean literal, remove type annotation.eslint[@typescript-eslint/no-inferrable-types](https://typescript-eslint.io/rules/no-inferrable-types)`